combat-trainer.lic

```diff
@@ -2186,6 +2188,20 @@ class CombatTrainer
     ]
   end
 
+  def get_process(process)
```

Comment:
I wonder if this is overly generic... would it be easier to just expose a method like `current_box_count` on `CombatTrainer` and let it look up the count on `LootProcess`?

Reply:
I personally dont think so. Having a method like this really opens up combat-trainer and allows us to interact with it at a 'super user' level. 

However, having more specific getter style methods is a bit cleaner when analyzing other scripts that may be interacting. 

If your concerned with using the get_process method I can make the helper function.

Reply:
Maybe my bigger concern is that other scripts would have to know which process to query for what, rather than treating `combat-trainer` as a black box.

Regardless, it's not critical right now. We can wait and see a few use cases before deciding.